pages/api/utils/pagePropsHelpers.ts

```diff
@@ -41,12 +46,35 @@ export const enforceAdmin: GetServerSideProps<PagePropsWithSession> = async (
 };
 
 // Redirect back to login screen if user isn't logged in
-export const loadSession: GetServerSideProps<PagePropsWithSession> = async (
+export const ensureSessionAndAccountList: GetServerSideProps = async (
   context,
 ) => {
   const session = await getSession(context);
   if (!session?.user.apiToken) {
-    return loginRedirect(context);
+    return {
+      ...loginRedirect(context),
+      props: {},
```

Comment:
Why are you adding `props`? You can either return a `redirect` or `props`, but shouldn't return both.